js/components/common/__tests__/InfoModalFields.test.js

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { render, screen, cleanup } from "@testing-library/react"
+import { Description } from "../react_modal.js.jsx"
+
+describe("Displays course information fields iff they are non-empty.", () => {
+  beforeEach(() => cleanup())
+
+  it("displays desc/prereq/coreq/excl/dist/breadth information if available", async () => {
+    const courseInfo = {
+      course: {
```

Comment:
You don't need the nested key here, just pass in `courseInfo` directly to the component (same as below)